apps/bare-expo/package.json

```diff
@@ -45,6 +45,7 @@
     "@react-native-picker/picker": "2.5.1",
     "@react-native-segmented-control/segmented-control": "2.4.1",
     "@shopify/flash-list": "1.6.3",
+    "@shopify/react-native-skia": "0.1.221",
```

Comment:
Is this change related to the update?

Reply:
ios won't build if we don't add the podspec.json. Needed to add it here to generate it. Temporary solution. Probably need to add something to et

Reply:
i think you could remove this after you copy the podspec into expo-go